src/extern/make.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python2.7
```

Comment:
yeah, this file is pretty....unique.  I'm fine with it for now, but it should not be in the `extern/` dir.  See below.